misk/src/test/kotlin/misk/web/interceptors/UserInterceptorTest.kt

```diff
@@ -52,15 +54,50 @@ class UserInterceptorTest {
     assertThat(response.body()!!.source()!!.readUtf8()).isEqualTo("text")
   }
 
-  internal class UserCreatedInterceptor: Interceptor {
+  @Test
+  fun throwResponse() {
+    val response = get("/call/throw")
+    assertThat(response.code()).isEqualTo(500)
+    assertThat(response.body()!!.source()!!.readUtf8()).isEqualTo("internal server error")
```

Comment:
The source shouldn’t be nullable. It might be, but it shouldn’t be

Reply:
Also `response.body()!!.string()` is a shortcut for `response.body()!!.source()!!.readUtf8()`

Reply:
IMO cleaner to do the final ``!!``just a the end

``response.body()?.source()?.readUtf8()!!``